Resources/doc/index.rst

```diff
@@ -1,15 +1,167 @@
-=============
-Configuration
-=============
+===========
+Quick start
+===========
 
-First of all you must config which shop you are going to work with: 
+Let's get started. We'll guide you step by step through installing and running ``ONGR sandbox`` integration with ``Magento`` for
+the first time. The first installation should not need more than 1.5 hour.
 
-.. code-block:: yaml
+Step 1: Requirements.
+---------------------
+
+Yes there are a few.
+
+Please check if your development environment in our handbook.
+
+Step 2: Download ONGR
+---------------------
+
+Download the latest release here `archive <https://github.com/ongr-io/ongr-sandbox/releases>`_ and unpack it somewhere under your project directory. Make sure that we have the "Vagrantfile" in the your project root folder.
+
+Checkout branch ``magento``
+
+Step 3: Install Virtual Box
+---------------------------
+
+Either install or upgrade `virtualbox <https://www.virtualbox.org/wiki/Downloads>`_. We need VirtualBox > 4.3
+
+Step 4: Install Vagrant
+-----------------------
+
+Either install or upgrade `vagrant <https://www.vagrantup.com/downloads.html>`_. We need Vagrant >= 1.6.5
+
+    (optional) Now we need to install the hosts updater vagrant plugin.
```

Comment:
It is either optional, or we need to install the hosts updater plugin. It can't be both.